### DISCUSSION TITLE
Upgrade timeline for Sveltekit 2 and later on Svelte 5

### DISCUSSION THREAD

**Discussion by hungryalgo** [huntabyte/shadcn-svelte#569] [Help]
When do we plan to upgrade to Sveltekit 2 and later on Svelte 5 when it is production ready? Do we have any rough timelines?

This would help us to plan our upgrades later. 

For this project with examples:
https://github.com/huntabyte/shadcn-svelte/tree/main/apps/www

I am referring to the package.json there:
https://github.com/huntabyte/shadcn-svelte/blob/main/apps/www/package.json

thanks!

**Comment by JamieWalkerSTF** [comment]
I'm using it in both Sveltekit 2 and Svelte 5 with no issues so far.
